src/ensembl/production/metadata/grpc/adaptors/genome.py

```diff
@@ -549,7 +544,7 @@ def fetch_genome_datasets(self, genome_id=None, genome_uuid=None, organism_uuid=
                     if release_version:
                         genome_select = genome_select.filter(EnsemblRelease.version <= release_version)
 
-            # print(f"genome_select str ====> {str(genome_select)}")
+            # print(f"fetch_genome_datasets query ====> {str(genome_select)}")
```

Comment:
Could we avoid keeping the `print` statements in PRs? May be good time to use loggers?